src/Manager.php

```diff
@@ -191,15 +196,19 @@ class RecursiveDirectoryFilterIterator extends \RecursiveFilterIterator
     /** @var array */
     private $excluded = array();
 
+    /** @var array */
+    private $OnlyFilesChanged = array();
```

Comment:
What about use other name for this class variable? What about something like `$changedFiles`?